### ISSUE TITLE
Consider removing EntityRole enumeration

### ISSUE THREAD

**Issue by cwhanse** [Open-Orange-Button/Orange-Button-Taxonomy#95]
Would have to create objects for the current enumerations.

**Comment by kevinkirkl3y** [comment]
@cwhanse there are instances where EntityRole is being used, see: Insurer, Preparer.

**Comment by cwhanse** [comment]
In those two cases, the object name has the meaning that the EntityRole value provides. I think our decision was to drop EntityRole from those two objects.

**Comment by kevinkirkl3y** [comment]
@cwhanse I don't believe so, @jrippingale had me add EntityRole specifically to those two elements, both of with are not values contained in the EntityRole enum list.
